treeherder/model/models.py

```diff
@@ -475,25 +475,9 @@ class Job(models.Model):
     objects = JobManager()
 
     id = models.BigAutoField(primary_key=True)
-
-    PENDING = 0
-    CROSSREFERENCED = 1
-    AUTOCLASSIFIED = 2
-    SKIPPED = 3
-    FAILED = 255
-
-    AUTOCLASSIFY_STATUSES = (
-        (PENDING, 'pending'),
-        (CROSSREFERENCED, 'crossreferenced'),
-        (AUTOCLASSIFIED, 'autoclassified'),
-        (SKIPPED, 'skipped'),
-        (FAILED, 'failed'),
-    )
-
     repository = models.ForeignKey(Repository, on_delete=models.CASCADE)
     guid = models.CharField(max_length=50, unique=True)
     project_specific_id = models.PositiveIntegerField(null=True)  # unused, see bug 1328985
-    autoclassify_status = models.IntegerField(choices=AUTOCLASSIFY_STATUSES, default=PENDING)
 
     # TODO: Remove coalesced_to_guid next time the jobs table is modified (bug 1402992)
     coalesced_to_guid = models.CharField(max_length=50, null=True, default=None)
```

Comment:
Is it worth removing `project_specific_id` and `coalesced_to_guid` while you're here?

Reply:
Great point, yeah.  I'll do that.